src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -207,6 +207,24 @@ double surface_integral_of_scalar(
     const Scalar<DataVector>& area_element, const Scalar<DataVector>& scalar,
     const Strahlkorper<Frame>& strahlkorper) noexcept;
 
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Euclidean surface integral of a vector on a 2D `Strahlkorper`
+ *
+ * \details Computes the surface integral \f$\oint V^i s_i d^2S\f$ for a
```

Comment:
Might it be clearer to just write the formula representing the value computed, including the normalization factor in the denominator?